### ISSUE TITLE
tuples on events

### ISSUE THREAD

**Issue by gilescope** [sjkillen/libspnav-rust#1]
rather than
```
MotionEvent {
                x,
                y,
                z,
                rx,
                ry,
                rz,
                period,
            }
```
it might be nice to do 
```
MotionEvent {
                (x, y, z),
                (rx, ry, rz),
                period,
            }
```
As tuples are structurally type matching someone might be able to use the translation `(x,y,z)` or the rotation as is without needing to enumerate the components.

**Comment by gilescope** [comment]
(also thank you - this lib works very nicely!)

**Comment by gilescope** [comment]
(Ah these comments are meant for the safe lib on top of this one.)

**Comment by sjkillen** [comment]
Good idea :) I've added some convenience methods that extract the 3 tuples from a motion event. You've also made me realize that, by error, many of the URLs point to this repository instead of https://github.com/sjkillen/spacenav-plus
Happy spacemousing!
